ydb/docs/en/core/changelog-server.md

```diff
@@ -1,5 +1,31 @@
 # {{ ydb-short-name }} Server changelog
 
+## Version 23.3.17 {#23-3-17}
+
+Release date: January 5, 2024.
+
+**Bugfixes:**
+
+* Fix `InFlightTxCount` become negative during `split/merge`.
+* Allow compatibility between non-stable versions of different applications.
+* Various stability improvements (`segfault`, handle balancer and boot queue events in `StateInit`, data race when reporting issue counters, memory leak in TRopeArena).
+* CPU over utilization in case of numerous topic consumers.
+* Fix import multiple converted csv columns.
+* Fix mem table merge when writing over committed deltas in tablets.
```

Comment:
it's not clear how severe this is — were merges before the fix leading to incorrect data stored and responded?